tools/changelog-helper.sh

```diff
@@ -44,9 +44,15 @@ find_or_add_missing_entries() {
         target_ref="${target_release_tag}"
```

Comment:
Should this be `local`?

Also, I see here we're fishing in `ChangeLog` for the version and asserting _that_ uses the `\d+\.`... format (unlike the tag).  All of this stuff needs documenting clearly in https://jamulus.io/contribute/Release-Process

Reply:
> Should this be `local`?

Yes. Updated (although unrelated to the changes here).

> Also, I see here we're fishing in `ChangeLog` for the version and asserting _that_ uses the `\d+\.`... format (unlike the tag). All of this stuff needs documenting clearly in https://jamulus.io/contribute/Release-Process
> 
> ("clearly" explaining why certain things should be done, so that if they need changing, how to change _this script_ (and others like it) can be understood.)

Yes, all of the versioning details would benefit from clear specifications and maybe some overhauls as well.